sycl/ReleaseNotes.md

```diff
@@ -16,6 +16,699 @@
+
+- Updated the project's
+  [security policy](https://github.com/intel/llvm/blob/b23d69e2c3fda1d69351137991897c96bf6a586d/SECURITY.md)
+  . intel/llvm#16559
+- Documented
+  [process](https://github.com/intel/llvm/blob/b23d69e2c3fda1d69351137991897c96bf6a586d/sycl/doc/developer/KHRExtensions.md)
+  of prototyping KHR extensions. intel/llvm#16883
+- Documented
+  [process](https://github.com/intel/llvm/blob/b23d69e2c3fda1d69351137991897c96bf6a586d/sycl/doc/developer/WorkingOnAReleaseBranch.md)
+  of working on release branches. intel/llvm#17042
+- Refreshed
+  [documentation](https://github.com/intel/llvm/blob/b23d69e2c3fda1d69351137991897c96bf6a586d/sycl/test-e2e/README.md)
+  on adding tests to the repository to reflect recent infrastructure
+  advancements/changes. intel/llvm#16409, intel/llvm#16875, intel/llvm#16967
+
+### Support for new hardware
+
+- Updated
+  [`sycl_ext_oneapi_device_architecture`](https://github.com/intel/llvm/blob/b23d69e2c3fda1d69351137991897c96bf6a586d/sycl/doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc)
+  extension specification and implementation to recognize Intel Panther Lake
+  H & U GPUs and Intel Xeon processors codenamed Diamond Rapids devices.
+  intel/llvm#16294, intel/llvm#16543
+- Taught the compiler about optional features supported by Intel Panther Lake
+  H & U GPUs (necessary for the correct AOT compilation). intel/lvm#16368
+- Updated
+  [`sycl_ext_intel_matrix`](https://github.com/intel/llvm/blob/b23d69e2c3fda1d69351137991897c96bf6a586d/sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_intel_matrix.asciidoc)
+  extension specification and implementation to support Intel Xeon processors
+  codenamed Diamond Rapids. intel/llvm#16543
+
+### Optimizations of SYCL Runtime
+
+Within this release some work has been done to reduce overheads incurred by
+SYCL runtime over low-level runtimes (such as Level Zero or OpenCL):
+
+- Reduced amount of string copies unnecessarily made by SYCL RT for debug traces
+  even if debug tracing is disabled. intel/llvm#16596
+- Reduced number of times `shared_ptr`s are copied. intel/llvm#17396,
+  intel/llvm#17477, intel/llvm#17473
+- Reduced amount of memory allocations happening by moving away from using
+  `std::function`. This should also help with reducing compilation time of SYCL
+  headers. intel/llvm#17202, intel/llvm#16668
+- Reduced amount of memory allocations required for `local_accessor`.
+  intel/llvm#17147, intel/llvm#17510
+- Reduce amount of memory allocations on "fast" kernel enqueue path and dropped
+  some unnecessary runtime checks. intel/llvm#17312, intel/llvm#17376
+- Made more queue operations go through "fast" path. intel/llvm#16735
+
+### Core SYCL 2020 functionality
+
+- Aligned `SYCL_LANGUAGE_VERSION` macro definition with the recent SYCL 2020
+  spec change (KhronosGroup/SYCL-Docs#704). intel/llvm#15890
+- Implemented `swizzle` method for swizzles. intel/llvm#16353
+
+### Other changes in SYCL Compiler
```

Comment:
Tagging @intel/dpcpp-clang-driver-reviewers, @intel/llvm-reviewers-cuda, @intel/dpcpp-tools-reviewers, @intel/unified-runtime-reviewers, tagging you to have a glance over this generic section

Reply:
Internal feedback from @maarquitos14 (@intel/dpcpp-tools-reviewers) : LGTM